kumascript/macros/LearnSidebar.ejs

```diff
@@ -1131,143 +1131,144 @@ var text = mdn.localStringMap({
         'What_is_JavaScript': '什么是 JavaScript？',
         'A_first_splash_into_JavaScript': '初次接触 JavaScript',
         'What_went_wrong': '出现了什么问题？JavaScript 疑难解答',
-        'Storing_the_information_you_need': '储存你所需的信息 — 变量',
-        'Basic_math_in_JavaScript': 'JavaScript 基础概念 — 数字与运算符',
-        'Handling_text': '处理文字 — JavaScript 中的字符串',
+        'Storing_the_information_you_need': '储存你所需的信息——变量',
+        'Basic_math_in_JavaScript': 'JavaScript 基础概念——数字与运算符',
+        'Handling_text': '处理文本——JavaScript 中的字符串',
         'Useful_string_methods': '有用的字符串方法',
         'Arrays': '数组',
-        'Assessment_silly_story_generator': '作业：傻瓜故事产生器',
+        'Assessment_silly_story_generator': '测验：傻瓜故事产生器',
       'JavaScript_building_blocks': 'JavaScript 基础要件',
         'JavaScript_building_blocks_overview': 'JavaScript 基础要件概述',
-        'Making_decisions_in_your_code': '在代码中决策 — 条件',
+        'Making_decisions_in_your_code': '在代码中决策——条件',
         'Looping_code': '代码循环',
         'Functions': '函数 - 可重复利用的代码块',
         'Build_your_own_function': '建立自己的函数',
-        'Function_return_values': '函数回传值',
+        'Function_return_values': '函数返回值',
         'Introduction_to_events': '事件介绍',
-        'Assessment_image_gallery': '作业：图库',
+        'Assessment_image_gallery': '测验：图库',
       'Introducing_JavaScript_objects' :  'JavaScript 对象介绍',
         'Introducing_JavaScript_objects_overview' :  'JavaScript 对象概述',
         'Object_basics' : '对象基础概念',
         'Object_prototypes' : '对象原型',
-        'Object-oriented_programming' : 'Object-oriented programming concepts',
-        'Classes_in_JavaScript' : 'Classes in JavaScript',
-        'Working_with_JSON_data' : '使用 JSON 资料',
+        'Object-oriented_programming' : '面向对象编程基本概念',
+        'Classes_in_JavaScript' : 'JavaScript 类',
+        'Working_with_JSON_data' : '使用 JSON 数据',
         'Object_building_practice' : '对象构建实践',
-        'Assessment_Adding_features_to_our_bouncing_balls_demo' : '作业：为弹跳球展示新增功能',
-      'Asynchronous_JavaScript': '异步JavaScript',
-        'Introducing_asynchronous_JavaScript': 'Introducing asynchronous JavaScript',
-        'How_to_use_promises': 'How to use promises',
-        'Implementing_a_promise-based_API': 'Implementing a promise-based API',
-        'Introducing_workers': 'Introducing workers',
-        'Assessment_sequencing_animations': 'Assessment: sequencing animations',
-      'Client-side_web_APIs': '客户端网页 API',
-        'Client-side_web_APIs_Introduction': '网页 API 介绍',
-        'Client-side_web_APIs_Manipulating_documents': '操纵文档',
+        'Assessment_Adding_features_to_our_bouncing_balls_demo' : '测验：为弹跳球展示新增功能',
+      'Asynchronous_JavaScript': '异步 JavaScript',
+        'Asynchronous_JavaScript_overview':'异步 JavaScript 概述',
+        'Introducing_asynchronous_JavaScript': '异步 JavaScript 简介',
+        'How_to_use_promises': '如何使用 promise',
+        'Implementing_a_promise-based_API': '实现基于 promise 的 API',
+        'Introducing_workers': 'workers 简介',
```

Comment:
```suggestion
        'Introducing_workers': 'worker 简介',
```

Reply:
@yin1999 We should update the context in some time.